frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, H6, Intent } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { navigate } from "@reach/router";
+import React, { FC, useState } from "react";
+import { ROUTES } from "src/common/constants/routes";
+import { Collection } from "src/common/entities";
+import { useDeleteCollection } from "src/common/queries/collections";
+import Alert from "src/components/Alert";
+
+interface Props {
+  id: Collection["id"];
+}
+
+const DeleteCollection: FC<Props> = ({ id }) => {
+  const [deleteMutation] = useDeleteCollection(id);
+
+  const handleDelete = async () => {
+    await deleteMutation(id);
+    navigate(ROUTES.MY_COLLECTIONS);
```

Comment:
ah actually now thinking about it, `await deleteMutation(id);` doesn't guarantee `deleteMutation()` succeeds, because React Query catches the error already, so `deleteMutation()` won't throw. 

So we might need to use:

```ts
import { useNavigate } from "@reach/router";
...
const [deleteMutation, { isSuccess }] = useDeleteCollection(id);
const navigate = useNavigate()

if (isSuccess) {
  navigate(ROUTES.MY_COLLECTIONS)
}
```

Reply:
@seve just wanted to double check you saw this 😆 Thank you!

Reply:
Oh! Totally right, I'll implement this now